internal/operators/ocs/validations.go

```diff
@@ -228,13 +232,16 @@ func (o *operator) setStatusInsufficientResources(nodeInfo *resourceInfo, mode o
 }
 
 // count all disks of drive type ssd or hdd
-func getValidDiskCount(disks []*models.Disk) int {
-	var countDisks int
-
+func getValidDiskCount(disks []*models.Disk) (int64, error) {
+	var countDisks int64
+	var err error
 	for _, disk := range disks {
-		if disk.SizeBytes >= conversions.GbToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
+		if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && !disk.Bootable {
+			if disk.SizeBytes < conversions.GbToBytes(MinDiskSize) {
+				err = fmt.Errorf("OCS Invalid Disk Size all the disks present in the cluster should be more than %d GB", MinDiskSize)
```

Comment:
@nbalacha  does the above message looks good?

Reply:
Lets use this for now and relook at all the messages in a separate PR.